index.html

```diff
@@ -0,0 +1,105 @@
+    <link rel="stylesheet" href="./css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://pegasus-ui.netlify.app/css/main.css">
+    <title>ShoeStore | HomePage </title>
+</head>
+<body>
+    <main>
+        <div class="big-wrapper">
+            <header>
+                <div class="container">
+                    <div class="logo">
+                        <h3>ShoeStore</h3>
+                    </div>
+                    <div class="links">
+                        <ul>
+                           <li><a href="https://github.com/RahulSingh9131" target="_blank">Github</a></li>
+                           <li><a href="https://twitter.com/singhrahul3112" target="_blank">Twitter</a></li>
+                           <li><a href="https://www.linkedin.com/in/rahul-singh-06b83917a/" target="_blank">LinkedIn</a></li>
+                           <li><a href="#" class="btn">LogIn</a></li>
+                        </ul>
+                    </div>
+                    
+                </div>
+            </header>
+            <section class="showcase-area">
+                <div class="container">
+                    <div class="left-side">
+                        <div class="big-title">
+                            <h1>Exclusive Store for only Footwear</h1>
+                            <h1>Start exploring now</h1>
+                        </div>
+                        <p class="text">
+                            ShoeStore contains a wide range of footwear products with multiple categories. SignUp to explore the store.
+                        </p>
+                        <div class="cta">
+                            <a href="#" class="btn">SignUp</a>
+                        </div>
+                    </div>
+                    <div class="right-side">
+                        <img src="./assets/heroImage.png" alt="" class="hero-image">
+                    </div>
+                </div>
+            </section>
+            <section class="category">
+                <div class="container">
+                    <h1 class="heading">
+                        <span>c</span>
+                        <span>a</span>
+                        <span>t</span>
+                        <span>e</span>
+                        <span>g</span>
+                        <span>o</span>
+                        <span>r</span>
+                        <span>i</span>
+                        <span>e</span>
+                        <span>s</span>
+                    </h1>
+                    <div class="box-container">
+                        <div class="box">
+                            <img src="https://images.unsplash.com/photo-1552422554-0d5af0c79fc6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
```

Comment:
we should give ``alt`` to every image it helps the screen reader to describe the image.